wheel/python/clvm_rs/chia_dialect.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class Dialect:
+    KEYWORDS: List[str]
+
+    NULL: bytes
+    ONE: bytes
+    TWO: bytes
+    Q_KW: bytes
+    A_KW: bytes
+    C_KW: bytes
+
+
+CHIA_DIALECT = Dialect(
+    (
+        # core opcodes 0x01-x08
+        ". q a i c f r l x "
+        # opcodes on atoms as strings 0x09-0x0f
+        "= >s sha256 substr strlen concat . "
+        # opcodes on atoms as ints 0x10-0x17
+        "+ - * / divmod > ash lsh "
+        # opcodes on atoms as vectors of bools 0x18-0x1c
+        "logand logior logxor lognot . "
+        # opcodes for bls 1381 0x1d-0x1f
+        "point_add pubkey_for_exp . "
+        # bool opcodes 0x20-0x23
+        "not any all . "
+        # misc 0x24
+        "softfork "
+    ).split(),
```

Comment:
I recall that `index` - `keyword` mapping like this is defined in `src/chia_dialect.rs`.
For maintenanceability, the mapping should be defined in one place.

Reply:
I agree that this violates "DRY". The problem is that in `src/chia_dialect.rs` all the integer to operator look-ups are done through symbols, which don't live past the compilation phase, so there's no way to actually export those symbols to python without some serious changes. Originally there was an `operator_lookup` table that was passed into the python version of clvm, and for a while there was even a version that got passed into rust, but Arvid was not a big fan and eventually replaced it with this static table.

Putting it back in would be a lot of work with not a clear gain, especially since `clvm_tools` is deprecated (and it was the main user of `operator_lookup`, adding `com` and `opt` among others). So I think this is the unfortunate compromise we're stuck with for now.

The main reason for exporting these is for use in a `disassemble` function (which is not included in this repo).